src/clients/mod.rs

```diff
@@ -84,16 +59,28 @@ mod test {
             ..Default::default()
         };
 
-        let (auth, value) = bearer_auth(&tok);
-        assert_eq!(auth, "authorization");
-        assert_eq!(value, "Bearer access_token");
+        let headers = tok.auth_headers();
+        assert_eq!(headers.len(), 1);
+        assert_eq!(
+            headers.get("authorization"),
+            Some(&"Bearer access_token".to_owned())
+        );
     }
 
     #[test]
     fn test_basic_auth() {
-        let (auth, value) = basic_auth("ramsay", "123456");
-        assert_eq!(auth, "authorization");
-        assert_eq!(value, "Basic cmFtc2F5OjEyMzQ1Ng==");
+        let creds = Credentials::new_pkce("ramsay");
```

Comment:
I think this test should be moved into `lib.rs`, the same file with `struct Credentials`

Reply:
Will do